src/PrestaShopBundle/Resources/config/services/core/grid/doctrine_query_builder.yml

```
@@ -456,3 +456,11 @@ services:
     autowire: true
     arguments:
       - '@prestashop.core.query.doctrine_search_criteria_applicator'
+
+  prestashop.core.grid.query_builder.customer_groups:
+    class: 'PrestaShop\PrestaShop\Core\Grid\Query\CustomerGroupsQueryBuilder'
```

```suggestion
  PrestaShop\PrestaShop\Core\Grid\Query\CustomerGroupsQueryBuilder:
```
We favor the FQCN notation now, for old definitions it may be complicated or long to migrate them at each bug fix related, but for all the new service definitions there's no reason not to do it the proper way the first time it will avoid adding an alias later